specification/apimanagement/resource-manager/readme.enable-multi-api.md

```diff
@@ -0,0 +1,102 @@
+# Multi-API support for AutoRest v3 generators
+
+``` yaml $(enable-multi-api)
+input-file:
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimanagement.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimapis.json
+  - >-
```

Comment:
Why these breaks `>`

Reply:
I believe, it's still valid YAML file. Probably, the line is too long. The file is automatically generated from existing `readme.md` files.